zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -205,7 +205,7 @@ public ZeppelinServer() throws Exception {
     this.interpreterService = new InterpreterService(conf, interpreterSettingManager);
 
     // Register MBean
-    if ("true".equals(System.getenv("ZEPPELIN_ENABLE_JMX"))) {
+    if ("true".equals(System.getenv("ZEPPELIN_JMX_ENABLE"))) {
```

Comment:
It looks like we didn't put it zeppelin-env.sh.template

Reply:
@zjffdu It was a typo in system variable name defined in zeppelin-env.sh.template.